openvpn/middlewares/client/bytescount/selective_stats_handler.go

```diff
@@ -0,0 +1,22 @@
+package bytescount
+
+import (
+	"github.com/pkg/errors"
+	"time"
+)
+
+// NewIntervalStatsHandler creates and returns composite handler, which invokes internal handler at given interval
+func NewIntervalStatsHandler(handler SessionStatsHandler, clock func() time.Time, interval time.Duration) (SessionStatsHandler, error) {
```

Comment:
clock -> currentTime ? It would look nicer now:=currentTime()

Reply:
Done.